tailcfg/tailcfg.go

```diff
@@ -1730,6 +1731,12 @@ const (
 
 	// CapabilityWarnFunnelNoHTTPS indicates HTTPS has not been enabled for the tailnet.
 	CapabilityWarnFunnelNoHTTPS = "https://tailscale.com/cap/warn-funnel-no-https"
+
+	// CapabilityTailnetLockAlpha indicates the node is in the tailnet lock alpha,
```

Comment:
wrong section. These are the warnings.  Put this up in the "Node Attributes" section (under line 1701/1702)

Reply:
Doneskie